backend/controllers/EventParticipation.js

```diff
@@ -1,26 +1,36 @@
-const Event = require("../models/Event");
-const EventParticipation = require("../models/EventParticipation");
-const User = require("../models/User");
-const { invite } = require("../utils/invite");
+const Event = require('../models/Event');
+const EventParticipation = require('../models/EventParticipation');
+const User = require('../models/User');
+const { invite } = require('../utils/invite');
 
-// invite function
+// Invite function
 const inviteUsers = async (req, res) => {
   const { invited } = req.body;
```

Comment:
NIT: I think this param name is confusing. From reading your code, this should be renamed to something like invitedUserEmails